core/src/test/scala/integration/kafka/api/LogDirFailureTest.scala

```diff
@@ -42,7 +40,6 @@ class LogDirFailureTest extends IntegrationTestHarness {
   this.logDirCount = 2
   this.producerConfig.setProperty(ProducerConfig.RETRIES_CONFIG, "0")
   this.producerConfig.setProperty(ProducerConfig.RETRY_BACKOFF_MS_CONFIG, "100")
-  this.serverConfig.setProperty(KafkaConfig.ReplicaHighWatermarkCheckpointIntervalMsProp, "100")
```

Comment:
We are changing this so that it's easier to trigger the issue, correct? Do we want to remove it altogether or make it higher than before but still lower than the default?